packages/eslint-plugin/tests/eslint-rules/no-restricted-globals.test.ts

```diff
@@ -30,6 +30,11 @@ type Handler = (event: string) => any
       `,
       options: ['event'],
     },
+    {
+      code: `
+        const a = foo?.bar?.name ?? "foobar"
```

Comment:
Prefer a separate test for nullish coalescing and optional chaining
```suggestion
        const a = foo?.bar?.name
```

Reply:
Done